components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+@protocol MDCBannerViewDelegate <NSObject>
+
+- (void)didTapButtonOnMDCBannerView:(UIButton *)button;
+
+@end
+
+@interface MDCBannerView : UIView
+
+@property(nonatomic, copy) NSString *text;
+
+@property(nonatomic, strong, nullable) UIImage *image;
+
+@property(nonatomic, readwrite, weak) id<MDCBannerViewDelegate> delegate;
```

Comment:
readwrite is the default, you can remove this annotation.

Reply:
removed:)